src/utils/global/tooltip/tooltip.module.scss

```
@@ -0,0 +1,39 @@
+@import '../../../variables.scss';
+
+.sharedStyles {
+  background-color: $tooltip-background-color;
+  border-radius: 0.5rem;
+  color: white;
+  font-size: 1rem;
+  left: calc(100% + .75rem);
+  padding: 0.5rem;
+  position: absolute;
```

```suggestion
  pointer-events: none;
  position: absolute;
```
I think you probably want to set `pointer-events: none` on the tooltip -- currently, if you hover where the tooltip will be, it will show it.